src/resources/js/views/rsvp.js

```diff
@@ -118,8 +111,23 @@
 	 *
 	 * @return void
 	 */
-	function tribe_rsvp_loader_end( ticket ) {
-		ticket.find( '.tribe-block__rsvp__loading' ).hide();
+	function tribe_rsvp_loader_end( $ticket ) {
+		$ticket.find( '.tribe-block__rsvp__loading' ).hide();
+	}
+
+	/**
+	 * Validates the RSVP form
+	 */
+	function validate_submission( $form ) {
+		var $qty = $form.find( 'input.tribe-tickets-quantity' );
+		var $name = $form.find( 'input.tribe-tickets-full-name' );
+		var $email = $form.find( 'input.tribe-tickets-email' );
+
+		return (
+			$.trim( $name.val() ).length
+				&& $.trim( $email.val() ).length
```

Comment:
Odd indentation.

Reply:
this is intentional, multi-line statements get a second indent. you'll see it happen in `bind_events()` too